apps/emqx_management/test/emqx_mgmt_cli_SUITE.erl

```diff
@@ -255,15 +255,15 @@ t_authz(_Config) ->
     {ok, _} = emqtt:connect(C),
     {ok, _, _} = emqtt:subscribe(C, <<"topic/1">>, 1),
     [Pid] = emqx_cm:lookup_channels(ClientIdBin),
-    ?assertMatch([_], gen_server:call(Pid, list_authz_cache)),
+    ?retry(100, 10, ?assertMatch([_], gen_server:call(Pid, list_authz_cache))),
```

Comment:
I also investigated this case a bit, but couldn't quickly figure out the reason why we have empty here. `emqtt:subscribe` is synchronous and populates the cache, how can we get empty 🤔 🤔 🤔

Reply:
Good question.  :thinking:

Reply:
Yep, adding a 1 ms sleep seems to stabilize the test.